### ISSUE TITLE
opensuse-tumbleweed build is failing continuously since 2020-03-09

### ISSUE THREAD

**Issue by mr-salty** [googleapis/google-cloud-cpp#3464]
example: https://source.cloud.google.com/results/invocations/0de0824b-99d6-46a6-b7d2-0438502e4d64/targets

notes ftom @coryan 
You are the operator this week right?  This is a different failure from #3437, so I am not nothing to investigate any further than what you will find below.  It is really hard to see what is going on because it is "the case of the dog that did not bark", the end of the log reads:

```
================================================================
Run Bigtable test programs against installed libraries opensuse-tumbleweed.
Created table tbl-1583871465-31780
bulk mutation successful
row-key-0: fam:c0 = v0
row-key-0: fam:c1 = v1
row-key-1: fam:c0 = v2
row-key-1: fam:c1 = v3
Deleted table tbl-1583871465-31780
Run Storage test programs against installed libraries opensuse-tumbleweed.
[ID: 4601247] Build finished after 1015 secs, exit value: 139
```

Okay, so a failure status, but what failed?  That last message reads like this on success (it does succeed when running locally):

```
Run Storage test programs against installed libraries opensuse-tumbleweed.
Successfully created object, generation=1583871862928949
================================================================
```

So the test program for storage:

https://github.com/googleapis/google-cloud-cpp/blob/master/ci/test-install/storage/storage_install_test.cc

failed, but printed nothing. That is all I know at this point.

**Comment by scotthart** [comment]
opensuse-tumbleweed is currently using libcurl version 7.69.0. Which is more recent than any other platform we test on. Manually testing with libcurl 7.69.1 reports an error of a bad parameter being passed to curl_easy_pause. By only checking to make sure the connection is paused before unpausing, the failure no longer occurs. While this appears to be the root cause, there is a possibility that the library tracking of closed_ could also contribute to the failing behavior.

**Comment by scotthart** [comment]
fixed in #3485 and #3487
